### ISSUE TITLE
How to laod pretrained  model

### ISSUE THREAD

**Issue by ForestWang** [dog-qiuqiu/Yolo-FastestV2#20]
I want to load the pretrained coco model when training. the coco ona has 80 classes. but mu one has only 2 classes.

so i the model created by Detector, the input parameter 'classes' is 2. so the model failed to load pretrained model. because the output tensor is not the same.

then, how to support laoding pretrained model?

**Comment by dog-qiuqiu** [comment]
可以参考model/backbone/shufflenetv2.py 代码，代码里面是默认加载预训练模型的

**Comment by data4pass** [comment]
What I did is to add "dummy" classes to match the amount of classes that the pre-trained weight has. So, for example. the pre-trained weight `coco2017-0.241078ap-model.pth` has 80 classes, but my dataset only has one class to detect. I added 79 dummy classes on my dataset's `.names` file, besides the one class that I actually want my model to detect.
